eclair-core/src/test/scala/fr/acinq/eclair/channel/states/StateTestsHelperMethods.scala

```diff
@@ -165,4 +162,7 @@ trait StateTestsHelperMethods extends TestKitBase {
 
   def channelId(a: TestFSMRef[State, Data, Channel]) = Helpers.getChannelId(a.stateData)
 
+  def feeConfOfChannel(a: TestFSMRef[State, Data, Channel]) = a.underlyingActor.nodeParams.onChainFeeConf
```

Comment:
Why not overload the name?

```suggestion
  def feeConf(a: TestFSMRef[State, Data, Channel]) = a.underlyingActor.nodeParams.onChainFeeConf
```

Reply:
And why didn't you make it return a `TestFeeEstimator`?

Reply:
Type erasure will not let you overload method names here, but returning a `TestFeeEstimator` or implemented a `setFee` method here would be cleaner

Reply:
Right, WDYT about using the pimp pattern here? `alice.feeEstimator`?

Reply:
I returned an `OnChainFeeConf` object because in some test we need both the estimator and the targets, the estimator is much more common though and i can return the `TestFeeEstimator` from the specialized helper method.

Reply:
Probably better to follow @sstone's recommendation and add a `setFeerate` method. You can still use `getFeeConf` when you need the targets.

Also the file isn't properly formatted.

Reply:
@araspitzu has updated with pimp pattern, looks nice IMO